targets/rpi4.docker

```diff
@@ -12,8 +12,8 @@ include(`modules/system.docker')dnl
 
 setdef(`__debootstrap_release__', `stable')dnl
 setdef(`__debootstrap_url__',     `http://deb.debian.org/debian/')dnl
-setdef(`__debootstrap_dinclude__', `gpg,dirmngr,debian-keyring,fancontrol,firmware-brcm80211,')dnl
-setdef(`__debootstrap_include__', `')dnl
+setdef(`__debootstrap_dinclude__', `gpg,dirmngr,debian-keyring,fancontrol,firmware-brcm80211,ca-certificates,multistrap')dnl
```

Comment:
I think we can remove `firmware-brcm80211` and perhaps even `__debootstrap_dinclude__` and `DEBOOTSTRAP_INCLUDE`.